### DISCUSSION TITLE
opsgenie elastalert - 401 Unauthorized for URL

### DISCUSSION THREAD

**Discussion by newcron** [jertel/elastalert2#474] [Q&A] [answered]
Hi Everyone, 

I'm trying to configure elastalert2 to alert to opsgenie. 

I've created an OpsGenie API Integration as described and have put the API key into the config under `opsgenie_key`. 

However, triggering an alert fails with that information in the logs:

```
2021-09-21 13:10:56,166     INFO           elastalert Error response from https://api.opsgenie.com/v2/alerts
 API Response: <Response [401]>
2021-09-21 13:10:56,166    ERROR           elastalert Error while running alert opsgenie: Error sending alert:
```

Has someone seen this before? debug logging also does not seem to help much - it gives me the payload that is sent to opsgenie - but not the HTTP headers. Any suggestion on what the issue might be?

**Comment by ferozsalam** [comment, marked as answer]
Are you able to use the API key to carry out operations successfully outside of ElastAlert? i.e. using `curl` or similar?

**Reply by newcron** [reply]
Thanks for the hint! Indeed, curl didn't work either. I also found the reason: I'm using the EU OpsGenie cluster which has a different API endpoint. after using `opsgenie_addr: https://api.eu.opsgenie.com/v2/alerts` in the config, alerts are working

**Reply by nsano-rururu** [reply]
You need to update the documentation.

https://docs.opsgenie.com/docs/alert-api
>If using the EU instance of Opsgenie, the URL needs to be https://api.eu.opsgenie.com for requests to be successful.
